tests/resolvers/Chat/admins.spec.ts

```diff
@@ -0,0 +1,40 @@
+import "dotenv/config";
+import { admins as adminsResolver } from "../../../src/resolvers/Chat/admins";
+import { connect, disconnect } from "../../helpers/db";
+import type mongoose from "mongoose";
+import { User } from "../../../src/models";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import type { TestChatType } from "../../helpers/chat";
+import { createTestChatMessage } from "../../helpers/chat";
+
+let testChat: TestChatType;
+let MONGOOSE_INSTANCE: typeof mongoose;
+
+beforeAll(async () => {
+  MONGOOSE_INSTANCE = await connect();
+  const userOrgChat = await createTestChatMessage();
+  testChat = userOrgChat[2];
```

Comment:
You access `testChat` using `userOrgChat[2]`. If the structure of `userOrgChat` changes or the function that generates it is modified, this can lead to bugs. Use destructuring to make your code more robust and readable.

```javascript
const [, , testChat] = await createTestChatMessage();
```